src/frontend/src/pages/GanttPage/GanttPackage/helpers/bar-helper.ts

```diff
@@ -33,12 +29,12 @@ export const convertToBarTasks = (
       barCornerRadius,
       handleWidth,
       rtl,
-      barProgressColor,
-      barProgressSelectedColor,
+      t.styles?.backgroundColor ?? '#9d9d9d',
+      t.styles?.backgroundColor ?? '#9d9d9d',
       barBackgroundColor,
       barBackgroundSelectedColor,
-      projectProgressColor,
-      projectProgressSelectedColor,
+      t.styles?.backgroundColor ?? '#e50000',
```

Comment:
same thing

Reply:
make the default be projectProgressColor